CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 NOTES:
 
+* This Go module has been updated to Go 1.18 per the Go support policy. Any consumers building on earlier Go versions may experience errors. (#445)
```

Comment:
Recommend either dropping this for provider Go upgrades, as its not relevant for practitioners, or tailoring the messaging for provider developers

```suggestion
```